ndcube/utils/cube.py

```diff
@@ -92,10 +92,13 @@ def _format_input_extra_coords_to_extra_coords_wcs_axis(extra_coords, missing_ax
         # of coord is same is data axis to which is corresponds.
         if coord[1] is not None:
             if not missing_axis[::-1][coord[1]]:
-
-                if len(coord[2]) != data_shape[coord[1]]:
-                    raise ValueError(coord_len_error.format(coord[0], len(coord[2]),
-                                                            data_shape[coord[1]]))
+                #if isinstance(coord[2], (list, tuple, np.ndarray)):
+                try:
```

Comment:
Add a comment here explaining why this ```try...except...``` statement is required and what it does.

Reply:
Yup, done.